cpp/ql/lib/semmle/code/cpp/ir/implementation/internal/TInstructionInternal.qll

```diff
@@ -2,3 +2,4 @@ import semmle.code.cpp.ir.internal.IRCppLanguage as Language
 import semmle.code.cpp.ir.implementation.raw.internal.IRConstruction as IRConstruction
 import semmle.code.cpp.ir.implementation.unaliased_ssa.internal.SSAConstruction as UnaliasedSsa
 import semmle.code.cpp.ir.implementation.aliased_ssa.internal.SSAConstruction as AliasedSSA
+import semmle.code.cpp.ir.implementation.aliased_ssa.internal.SideEffectElimination as Elim
```

Comment:
I don't think you've checked `SideEffectElimination.qll` in yet. That explains all the consistency errors in ql-for-ql.